src/app/app.config.ts

```diff
@@ -95,3 +95,13 @@ export const languageConfig = {
    */
   defaultLanguage: 'en'
 };
+
+/**
+ * List with named addresses.
+ */
+export const namedAddresses = [
+  {
+    address: '2iNNt6fm9LszSWe51693BeyNUKX34pPaLx8',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skycoin-explorer/pullRequest?prid=7163626)
```suggestion
    address: "2iNNt6fm9LszSWe51693BeyNUKX34pPaLx8",
```